src/administration/integrations/webhooks.md

```diff
@@ -11,7 +9,73 @@ launch automated user testing.
 $ platform integration:add --type=webhook --url=A-URL-THAT-CAN-RECEIVE-THE-POSTED-JSON
 ```
 
-Here you can see an example of the response:
+The webhook URL will receive a POST message for every "Activity" that is triggered, and the message will contain complete information about the entire state of the project at that time.  In practice most of the message can be ignored but is available if needed.  The most commonly used values are documented below.
+
+It's also possible to set the integration to only send certain activity types, or only activities on certain branches.  The CLI will prompt you to specify which to include or exclude.  Leave at the default values to get all events on all environments in a project.
+
+## Webhook schema
+
+### `id`
+
+A unique opaque value to identify the activity.
+
+### `project`
+
+The Project ID for which the activity was triggered.  Use this value if you want to have multiple projects POST to the same URL.
+
+### `type`
+
+The `type` property specifies the event that happened.  Its value is one of:
+
+* `environment.push`: A user has pushed code to a branch, either existing or new.
+* `environment.branch`: A new branch has been created via the UI. (A branch created via a push will show up only as an `environment.push`.)
+* `environment.backup`: A user triggered a [snapshot](/administration/snapshot-and-restore.md)
+* `environment.restore`: A user triggered a [snapshot](/administration/snapshot-and-restore.md)
+* `environment.subscription.update`: I don't know what this is.
+* `environment.activate`: A branch has been "activated", and an environment created for it.
+* `environment.deactivate`: A branch has been "deactivated". The code is still there but the environment was destroyed.
+* `environment.synchronize`: An environment has had its data re-copied from its parent environment.
+* `environment.delete`: A branch was deleted.
+* `environment.merge`: A branch was merged through the UI or Platform.sh API. A basic Git merge will not trigger this event.
+* `environment.merge-pr`: A branch was merged via an accepted Pull Request on GitHub. 
```

Comment:
Is merge-pr also fired for GitLab/Bitbucket PRs?

Reply:
That's currently only used for Bitbucket, and it is probably going away. Other integrations are using `environment.push` directly: it's used when Bitbucket updates a PR environment by merging the PR branch into the target branch.

Reply:
OK, I'll remove that one outright.